playback/tape_recorder.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, tape_cassette):
         self._playback_outputs = []
         self._invoke_counter = Counter()
         self._classes_recording_params = {}
-        self._random = Random(110613)
+        self._random = Random()
```

Comment:
Better to pass seed as an argument to the constructor with default value None and in the test pass a specific value instead of replacing private member in the test